x-pack/filebeat/input/salesforce/input.go

```diff
@@ -181,15 +187,18 @@ func (s *salesforceInput) isError(err error) error {
 }
 
 func (s *salesforceInput) SetupSFClientConnection() (*soql.Resource, error) {
+	s.log.Info("Setting up Salesforce client connection")
 	if s.sfdcConfig == nil {
 		return nil, errors.New("internal error: salesforce configuration is not set properly")
 	}
 
 	// Open creates a session using the configuration.
 	session, err := session.Open(*s.sfdcConfig)
 	if err != nil {
+		s.log.Errorf("Failed to open Salesforce session: %v", err)
```

Comment:
why log it here?
it would eventually logged in parent functions, right?

Reply:
yeah can do that. i'll improve the returning error message so that can identify which method returned the error.

Reply:
done!